### PR TITLE
feat: add option to give feedback from the website

### PR DESCRIPTION
**Pull request by burakukula**

### CODE REVIEW

File: packages/forma-36-website/src/pages/index.jsx [hunk 1]
```diff
@@ -64,6 +64,25 @@ function IndexPage() {
...
+          <TextLink
+            href="https://github.com/contentful/forma-36/issues/new/choose"
+            target="_blank"
+          >
+            GitHub
+          </TextLink>
+        </Paragraph>
+        <Paragraph>
+          or use this anonymous{' '}
+          <TextLink href="https://forms.gle/qC7LLbiy4CcF5HPLA" target="_blank">
```
Comment (massao) [inline packages/forma-36-website/src/pages/index.jsx:80]: shouldn't this also have the `rel="noopener noreferrer`?

**Review by massao — APPROVED** [review]
LGTM, just a small comment

**Review by thebesson — APPROVED** [review]
👏👏👏

File: packages/forma-36-website/src/pages/index.jsx (hunk 1 above)
Comment (burakukula) [reply packages/forma-36-website/src/pages/index.jsx:80]: good point! fixing

**Pull request merged**